### PR TITLE
Fix

### PR DESCRIPTION
**Pull request by kellerd**
Addresses interop with VB.NET #38

e7e1ff8 Adds `[<assembly: ExtensionAttribute()>]` to AssemblyInfo.fs, Adds AssemblyInfo.fs to Chessie.fsproj
7b9c724 Creates test projects for VB using code from the C# project
02ff8a9 Adds test project to solution, may want to exclude this one and do it manually as it seems my version Visual Studio is ahead of the project

### CODE REVIEW

File: Chessie.sln [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 2012
-VisualStudioVersion = 12.0.31101.0
+# Visual Studio 2013
```
Comment (forki) [inline Chessie.sln]: do we need that change?

Comment (kellerd) [inline Chessie.sln]: This is the part I wasn't sure of. I only have VS2013 and 2015 on this box. 
VS forced the change when I added the test project to the solution, kept as a separate commit.

Comment (keichinger) [inline Chessie.sln]: You don't need to necessarily commit certain lines. In almost every Git GUI Tool (if you aren't familiar with the Git CLI) you can select which lines you want and which lines you want to discard (or keep staged).

Comment (forki) [inline Chessie.sln]: It's not really important...

**Comment by forki** [comment]
could you please take a look at the travis/mono build?

**Comment by mexx** [comment]
LGTM, besides the travis/mono build error.

**Comment by kellerd** [comment]
After some investigation, and learning looks like the mono vb.net compiler is basically running Visual Basic 8 from Visual Studio 2005, with no plans to upgrade beyond that.

So things in the tests that have come since then, don't work:

Auto-Properties
Object Initialization using With
Lambdas
List Initialization using {}
Linq query syntax

I'll assume you guys would still want tests written, so I'll re-implement them old school style.

**Comment by kellerd** [comment]
Ok, need some suggestions here on how you guys would like me to proceed. 

Had a lapse, forgetting in 2005 we also didn't have Extension methods,  which is what this PR is specifically trying to enable for VB.  Mono has no support for it (in VB), and will likely never.

From what I've read quickly so far, baring some ugly hacks, the tests would go from:

Testing calling the extension methods in VB
to
Can I call an F# library from VB with some ugly hackish code.

Should I 
- Abandon PR
- Write tests as they should work (not MONO) and #if Not MONO to exclude the tests on mono
- Duplicate the number of tests, one for mono, one for not

**Comment by mexx** [comment]
I would vote for simply exclude the tests on mono, as VB support is that much limited on mono.

**Comment by forki** [comment]
Yes. Either somehow ifdef the stuff or just delete the Vb project
altogether.

**Comment by kellerd** [comment]
Removed the project. It builds, learned a lot from what started as a simple effort and proof of concept for work.

**Comment by forki** [comment]
thanks!

**Pull request merged**